moveit_core/collision_detection/include/moveit/collision_detection/collision_detector_allocator.h

```diff
@@ -64,8 +65,7 @@ namespace collision_detection
     virtual CollisionRobotPtr allocateRobot(const CollisionRobotConstPtr& orig) const = 0;
   };
 
-  typedef boost::shared_ptr<CollisionDetectorAllocator> CollisionDetectorAllocatorPtr;
-
+  MOVEIT_CLASS_FORWARD(CollisionDetectorAllocator);
```

Comment:
Will do. That actually makes it easier to verify that the correct name is passed to the macro :)